ksqldb-functional-tests/src/test/resources/historical_plans/protobuf_-_protobuf_nosr_primitives_with_null/7.4.0_1668095424764/plan.json

```diff
@@ -0,0 +1,214 @@
+{
```

Comment:
Why are there two new historic plan versions for this test in this PR? There should only be one, right? Same question for the other two PROTOBUF_NOSR tests in this PR.

Reply:
hmm you are right. Not sure why they were created